modules/ROOT/pages/x12-identity-settings.adoc

```diff
@@ -61,22 +61,22 @@ Refer to this information specifically for using the DUNS number as an identifie
 |Setting |Description |Required
 
 |Identifier Value
-|The DUNS number for your trading partner. The DUNS number is a nine-digit number that your trading partner needs to supply you with that corresponds to the Sender/Receiver ID.
+|The DUNS number for your partner. The DUNS number is a nine-digit number that your partner needs to supply you with that corresponds to the Sender/Receiver ID.
```

Comment:
The DUNS number is a nine-digit number that your partner provides that corresponds to the Sender/Receiver ID.

Reply:
OK